Add license information

oaustegard opened the pull request:
A prospective user should not have to trawl Github and the [AWS Service Terms](https://aws.amazon.com/service-terms/#55._AWS_Greengrass) in order to locate the license for each component.

*Issue #, if available:*

*Description of changes:*


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

oaustegard committed:
Add license information

A prospective user should not have to trawl Github and the [AWS Service Terms](https://aws.amazon.com/service-terms/#55._AWS_Greengrass) in order to locate the license for each component.

oaustegard committed:
Update public-components.md

Tweaked language to more closely match the Service Terms page

richa-aws commented:
Thanks for your pull request. We're reviewing how we address licensing information for AWS-provided components, and we'll provide updates here about progress.

richa-aws commented:
Hi @oaustegard, we're still reviewing how we present licensing information for components in our documentation. I'll provide additional updates as they're available.

richa-aws commented:
Hi @oaustegard, 
First of all, thank you for your initial pull request. I just published an update to the [open-source](https://github.com/awsdocs/aws-iot-greengrass-v2-developer-guide/blob/main/doc_source/open-source.md) topic that adds additional context around licensing for open source components, and for those components that are not made available under an open source license. 
I think this update addresses your feedback, so I'm going to close this pull request. We would, however, really appreciate if you review the updated content and submit any additional changes, if needed!
I've also added a task to the team's list of items for future consideration to add licensing information for individual components.

oaustegard commented:
Thanks! That is concise yet informative. 👍👍

On Fri, Jun 11, 2021 at 7:52 PM Richa Mehta ***@***.***>
wrote:

> Hi @oaustegard <https://github.com/oaustegard>,
> First of all, thank you for your initial pull request. I just published an
> update to the open-source
> <https://github.com/awsdocs/aws-iot-greengrass-v2-developer-guide/blob/main/doc_source/open-source.md>
> topic that adds additional context around licensing for open source
> components, and for those components that are not made available under an
> open source license.
> I think this update addresses your feedback, so I'm going to close this
> pull request. We would, however, really appreciate if you review the
> updated content and submit any additional changes, if needed!
> I've also added a task to the team's list of items for future
> consideration to add licensing information for individual components.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/awsdocs/aws-iot-greengrass-v2-developer-guide/pull/11#issuecomment-859961026>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAY6BAYHRYM6XFRCNJUW5JDTSKOSLANCNFSM45XLQZOQ>
> .
>
-- 
Oskar Austegard
